sample_workloads/lit-gpt-demo/slurm/litgpt_container.sh

```diff
@@ -0,0 +1,46 @@
+MASTER_ADDR=$(scontrol show hostnames "$SLURM_JOB_NODELIST" | head -n 1)
+
+# Start the Lit-GPT training container
+docker run \
+    --privileged \
+    --gpus all --net="host" \
+    -v /var/lib/tcpx/lib64:/var/lib/tcpx/lib64 \
+    -v ${UDS_PATH}:${UDS_PATH} \
+    -u 0 \
+    -e LD_LIBRARY_PATH=/usr/local/nvidia/lib64:/var/lib/tcpx/lib64:/usr/lib/lib32:/usr/lib/x86_64-linux-gnu/ \
+    -e JOB_TIMESTAMP=111113 \
```

Comment:
Probably should change this.

Maybe just set to epoch time.

-e JOB_TIMESTAMP=$(date +%s) \